### DISCUSSION TITLE
欢迎来到玉兔毫讨论区！

### DISCUSSION THREAD

**Discussion by amorphobia** [amorphobia/rabbit#2] [Announcements]
## 👋欢迎！
  您可以使用讨论区分享和获取使用玉兔毫的体验。您可以：
  * 提出您感兴趣的问题。
  * 分享您的想法。
  * 与其他用户交流。
  * 打开思路，集思广益。让我们共同创建这个社区💪。

**Comment by yzy-boop** [comment]
玉兔确实比rime简单，不用安装。
但右键“输入法设定”中的字体设置是灰色的，不能修改。好象只能通过修改rabbit.yaml。
也不知道从哪儿修改输入法图标，
及怎么修改哪儿能打出配对标点符号（如括号、双引号）
还有五笔拼音混输时，先出现拼音的字，不显示五笔码提示。
能否象小科狗的码表那样，增加一列，即4列：key	value	weight	fc，其中key是字词的码表，fc是提示（可以自定义，如五笔、拼音、拆码等）
